src/Symfony/Component/Process/Exception/ProcessRuntimeException.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Exception;
+
+use Symfony\Component\Process\Process;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class ProcessRuntimeException extends RuntimeException
+{
+    /**
+     * @var Process
```

Comment:
Do we need these PHPdoc now that we use type hints and return types?

Reply:
Probably not, indeed.

Reply:
Well, I maybe spoke to fast. The PHPdoc is for the property, not the constructor.

It could be remove from `getProcess` indeed, but I don't know for this one.

Reply:
You can remove it, the constructor already determines the type of this property. Your IDE will be able to detect this.